sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/ParameterLoader.cs

```diff
@@ -86,7 +86,7 @@ internal static class ParameterLoader
 
                 if (parameterValue == null)
                 {
-                    return;
```

Comment:
the logic didn't work if the rule had 2 parameters and the sonarlint.xml contained the second parameter only - this logic was stopping too early